src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.CompilationManager.cs

```diff
@@ -42,6 +42,9 @@ internal partial class DiagnosticAnalyzerService
             var compilationWithAnalyzersPair = await CreateCompilationWithAnalyzersAsync().ConfigureAwait(false);
             tupleBox = new((stateSets, compilationWithAnalyzersPair));
 
+#if NET
```

Comment:
this running out of proc or in-proc in VS? (curious if this only applies to VSCode)

Reply:
this runs in proc in VS currently.  but i would like it to run OOP.  basically, we run most of diagnostics *in proc* *until* the point of calling out to actually do the computation.  i'd like to push this much higher so that we basically remote over the call to the IDiagService immediately,  and do everything else on the OOP side.  that is upcoming.